source/draw/AbstractContext.ooc

```diff
@@ -36,4 +36,6 @@ AbstractContext: abstract class {
 	alignWidth: virtual func (width: Int, align := AlignWidth Nearest) -> Int { width }
 	update: abstract func
 	isAligned: virtual func (width: Int) -> Bool { true }
+	_createContext := static func -> This { CpuContext new() }
```

Comment:
Sure.
This design is actually Simon's idea, but unfortunately I could not have done it like
```ooc
_createContext := CpuContext new
```
as he originally suggested. I needed to wrap it in a closure here.